packages/dynamite/dynamite/lib/src/openapi_builder.dart

```diff
@@ -625,20 +626,24 @@ class OpenAPIBuilder implements Builder {
                                     if (mimeType == '*/*' || mimeType.startsWith('image/')) {
                                       dataType = 'Uint8List';
                                       dataValue = 'response.bodyBytes';
```

Comment:
Can't we add the await to the dataValue directly?

Similarly as it's already done in the cases where we don't need to explicitly await it.

Reply:
Ahhhh this was for a completely different reason. If you do your suggestion then you end up generating `return await response.bodyBytes;` which is a linting error and `dart fix` doesn't fix it for some reason. That's why the switch is needed to only add the await in specific places where it is actually needed. Does that make sense?